proxy/Main.cc

```diff
@@ -701,7 +703,13 @@ CB_After_Cache_Init()
 
   if (1 == start) {
     Debug("http_listen", "Delayed listen enable, cache initialization finished");
-    start_HttpProxyServer();
+    if (num_accept_threads == 0) {
```

Comment:
Why not a single `if` with `&&`?

Reply:
oh, I forgot to change that one, copied the old check over there, I'll change it with something else I found